dfc-content-pkg-netcore/Data/Enums/ContentCacheStatus.cs

```diff
@@ -5,5 +5,6 @@ public enum ContentCacheStatus
         Content = 0,
         ContentItem = 1,
         NotFound = 2,
+        Both = 3,
```

Comment:
I would switch this enum around then you can use bitwise operators later on?
NotFound=0
Content=1
ContentItem=1
Both = Content | ContentItem